.github/ISSUE_TEMPLATE/accessibility.yml

```diff
@@ -0,0 +1,81 @@
+name: Accessibility task
+description: |
+  Propose an accessibility improvement,
+  following WAVE report in https://github.com/jupyterhub/outreachy/issues/39.
+
+title: "Accessibility: "
+labels:
+  - outreachy-dec22
+  - "project: improve-accessibility"
+  - "needs:review"
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for improving accessibility of a Jupyter page!
+
+        If you're here, you have produced a WAVE accessibility report (as described in #39).
```

Comment:
```suggestion
        If you're here, you have produced a WAVE accessibility report (as described in https://github.com/jupyterhub/outreachy/issues/38).
```

Reply:
oops, no, thinking of soemthing else. This is better!